src/preferences/preferences.hpp

```diff
@@ -180,6 +181,15 @@ struct option
 };
 };
 
+using std::string;
```

Comment:
please don't do this in a global namespace in a header file. All files that include this file will now also have string in global namespace.